rel_scripts/entrypoint.sh

```diff
@@ -20,5 +20,5 @@ set -e
 ROOT="$(realpath $(dirname $0)/..)"
 RELEASE_DIR="${ROOT}/releases/{{release_version}}"
 YAML_CONFIG=${1:-${ROOT}/etc/config.yaml}
-ERL_LIBS="${ROOT}/lib" ${ROOT}/erts-{{release_erts_version}}/bin/escript ${ROOT}/bin/configurator.escript ${YAML_CONFIG} ${RELEASE_DIR}
+ERL_LIBS="${ROOT}/lib" /usr/local/bin/escript ${ROOT}/bin/configurator.escript ${YAML_CONFIG} ${RELEASE_DIR}
```

Comment:
А почему был выбран вызов по полному пути, а не
```suggestion
ERL_LIBS="${ROOT}/lib" escript ${ROOT}/bin/configurator.escript ${YAML_CONFIG} ${RELEASE_DIR}
```
?

Reply:
Потому что он там точно есть :) Впрочем в пути тоже есть, так что не принципиально наверное